storage/src/mmr/bitmap.rs

```diff
@@ -77,19 +94,104 @@ impl<H: CHasher, const N: usize> Bitmap<H, N> {
             bitmap,
             next_bit: 0,
             mmr: Mmr::new(),
+            pruned_chunks: 0,
+            pinned_nodes: Vec::new(),
         }
     }
 
-    /// Return the number of bitmap chunks that have been pruned.
-    #[inline]
-    fn pruned_chunks(&self) -> usize {
-        self.mmr.pruned_to_pos() as usize
+    const NODE_PREFIX: u8 = 0x00;
```

Comment:
nit: I prefer `const` outside of `impl`. Not sure what is more canonical/common in Rust.

Reply:
not aware of any common idioms here, so moved outside.

Reply:
Although the prefix vars for journaled.rs are also defined inside its impl.  I can pull those out too though?

Reply:
moved those out too.